src/_migrate.sh

```diff
@@ -147,14 +147,14 @@ function checkExistingConfigForOverrides() {
   esac
 
   info "Checking legacy configuration"
-  dcCMD="docker-compose -f $legacyComposeFile -f $legacyDatabaseFile"
+  dcCMD="docker compose -f $legacyComposeFile -f $legacyDatabaseFile"
   ${dcCMD} config -q || die "Invalid legacy configuration - please contact support"
 
   decodedComposeFile=$(base64 -d <<<$DOCKER_COMPOSE_ENCODED)
   #diff -N --unified=2 --color=always --label existing --label "updated" $targetComposeFile <(echo "$decodedComposeFile") || return 0
   diff --unified --color=always --show-function-line='^\s\{2\}\w\+' \
     <($dcCMD config --no-interpolate) \
-    <(docker-compose -f - <<< "${decodedComposeFile}" -f $composeOverrideFile config --no-interpolate) || return 0
+    <(docker compose -f - <<< "${decodedComposeFile}" -f ${composeOverrideFile} config --no-interpolate) || return 0
   return 1
   #diff --color=always -y --left-column <($dcCMD config --format=json | jq -S . -r) <(docker-compose -f - <<< "$decodedComposeFile" -f $composeOverrideFile config --format=json | jq -S . -r) | grep -v '^\+'
```

Comment:
Simultaneously early and late to the party, having started this work in a branch on my fork a long while back and never getting around to PR'ing it 😬 , and then I missed commenting before this one is merged.

The only difference I see in mine that appears missing in this one is that lines 84, 134 and 159 in `src/_migrate.sh` reference `docker-compose` in log/comments, in case you decide you want to go for consistency.

Reply:
Oh nice catch; will add that into the Bugfix PR